pom.xml

```diff
@@ -14,15 +14,15 @@
     <maven.compiler.target>8</maven.compiler.target>
     <maven.compiler.source>8</maven.compiler.source>
 
-    <io.jsonwebtoken.jjwt.version>0.11.2</io.jsonwebtoken.jjwt.version>
+    <io.jsonwebtoken.jjwt.version>0.12.3</io.jsonwebtoken.jjwt.version>
 
-    <junit.jupiter.version>5.8.2</junit.jupiter.version>
-    <assertj.core.version>3.22.0</assertj.core.version>
-    <mockito.core.version>4.3.1</mockito.core.version>
-    <findbugs.annotations.version>3.0.1</findbugs.annotations.version>
+    <junit.jupiter.version>5.10.1</junit.jupiter.version>
+    <assertj.core.version>3.24.2</assertj.core.version>
+    <mockito.core.version>4.11.0</mockito.core.version>
+    <findbugs.annotations.version>4.8.2</findbugs.annotations.version>
```

Comment:
You probably want to rename this to spotbugs, since it's not called findbugs anymore.